module/translation-deepl/src/Infrastructure/Provider/TranslationDeeplProviderInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See license.txt for license details.
+ */
+
+declare(strict_types = 1);
+
+namespace Ergonode\TranslationDeepl\Infrastructure\Provider;
+
+use Ergonode\Core\Domain\ValueObject\Language;
+
+/**
+ */
+interface TranslationDeeplProviderInterface
```

Comment:
I think this class should be called... Deepl is only one of the possible implementations